lib/content-services/src/lib/search/components/search-datetime-range/search-datetime-range.component.ts

```diff
@@ -133,18 +135,43 @@ export class SearchDatetimeRangeComponent implements SearchWidget, OnInit {
 
         this.setFromMaxDatetime();
         this.enableChangeUpdate = this.settings?.allowUpdateOnChange ?? true;
+        this.context.populateFilters
+            .asObservable()
+            .pipe(takeUntil(this.destroy$))
+            .subscribe((filtersQueries) => {
+                if (filtersQueries[this.id]) {
```

Comment:
same here like in previous file - can you put filtersQueries[this.id] in some const or use some rxjs operator which give you only that single value?